First take at READMEs (#86)
canonical/dns-operators: dns-integrator-operator/README.md

[![CharmHub Badge](https://charmhub.io/bind/badge.svg)](https://charmhub.io/dns-integrator)
[![Publish to edge](https://github.com/canonical/dns-operators/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/dns-operators/actions/workflows/publish_charm.yaml)
[![Promote charm](https://github.com/canonical/dns-operators/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/dns-operators/actions/workflows/promote_charm.yaml)
[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)

> # Dns Integrator Operator

Reviewer: ```suggestion
# DNS Integrator Operator
```

[![CharmHub Badge](https://charmhub.io/bind/badge.svg)](https://charmhub.io/dns-integrator)
[![Publish to edge](https://github.com/canonical/dns-operators/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/dns-operators/actions/workflows/publish_charm.yaml)
[![Promote charm](https://github.com/canonical/dns-operators/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/dns-operators/actions/workflows/promote_charm.yaml)
[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)

# Dns Integrator Operator

A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
deploying and managing a DNS record requests integrator on bare metal.

This charm simplifies the resource record requests creation towards an already deployed DNS charm
accepting the `dns_record` interface.

## Get started

> The charm can be deployed to any machine model by pulling it from [charmhub](https://charmhub.io/dns-integrator):

Reviewer: ```suggestion
The charm can be deployed to any machine model by pulling it from [Charmhub](https://charmhub.io/dns-integrator):
```

```
juju deploy dns-integrator --channel=latest/edge
```

You can then integrate it with any charm supporting the requirer side of the `dns_record` interface.  

> Use the configuration of the charm to decide which resource record requests you want to send through the integration.

Reviewer: ```suggestion
Use the configuration of the charm to decide which resource record requests you want to send to the requirer charm.
```
Let me know if that's not correct -- but my thought process is that it may be more valuable to focus on where you're sending the resource records requests to.

Author: I used your version as-is

> Use the configuration of the charm to decide which resource record requests you want to send through the integration.

Reviewer 2: Any mandatory configurations?

### Basic operations

No actions are available as this charm is meant to be operated through its integrations.  

> It can integrate with any requirer charm implementing the [dns_record interface](https://canonical.github.io/charm-relation-interfaces/interfaces/dns_record/v0/).

Reviewer: ```suggestion
The charm can integrate with any requirer charm implementing the [dns_record interface](https://canonical.github.io/charm-relation-interfaces/interfaces/dns_record/v0/).
```

## Learn more
* [Read more](https://charmhub.io/dns-integrator/docs)

> * [Official webpage](https://charmhub.io/dns-integrator)

Reviewer: It may be better to remove the "Official webpage" link here

## Project and community
* [Issues](https://github.com/canonical/dns-operators/issues)

> * [Contributing](https://charmhub.io/bind/docs/how-to-contribute)

Reviewer: ```suggestion
* [Contributing](https://github.com/canonical/dns-operators/blob/main/CONTRIBUTING.md)
```
